### ISSUE TITLE
not discovering devices

### ISSUE THREAD

**Issue by pavankumarupai37** [vitorpamplona/react-native-ble-advertiser#5]
hi 
i have enabled location too despite it is not showing any devices, i have 2 devices in range with bluetooth turned on.Is this issue related to android versions?

**Comment by vitorpamplona** [comment]
Hi there! 

Android seems to be functioning perfectly. Which models are you using? Are you just running the example code or your own? 

Best

**Comment by pavankumarupai37** [comment]
Yeah I'm just running the code...and I'm not able to discover any devices

**Comment by pavankumarupai37** [comment]
Hi
I have posted in issue section kindly help me out with the issue pls

**Comment by vitorpamplona** [comment]
What is the log showing?

**Comment by pavankumarupai37** [comment]
Should I configure anything else apart from switching on the location?

**Comment by vitorpamplona** [comment]
No, it's probably something wrong with the hardware support. Have you run through the Android Studio? It will show the LogCat messages (it shows each new device it finds). 

Which phones are you testing on?

**Comment by pavankumarupai37** [comment]
I'm testing on moto g5 s+

**Comment by vitorpamplona** [comment]
Oh, the g5 is not a Bluetooth 5.0 device. That could bring problems since I have not tested in these older Bluetooth versions. :(

**Comment by pavankumarupai37** [comment]
Oh I see it works only for Bluetooth 5.0 and above.I will surely work on this and try to submit you a fix asap.Thank you😀
